packages/sw-cli/src/lib/utils/generate-glob-pattern.js

```diff
@@ -0,0 +1,14 @@
+const path = require('path');
+
+module.exports = (relativePath, fileExtentionsToCache) => {
+  // Glob patterns only work with forward slash
+  // https://github.com/isaacs/node-glob#windows
+  const globPath = path.join(relativePath, '**', '*').replace(path.sep, '/');
+  if (fileExtentionsToCache.length > 1) {
+    // Return pattern '**/*.{txt,md}'
+    return globPath + `.{${fileExtentionsToCache.join(',')}}`;
+  } else {
+    // Return pattern '**/*.txt'
+    return globPath + `.${fileExtentionsToCache[0]}`;
```

Comment:
Very good point - it wasn't an issue for the CLI because I had a validation check, but for node module - yes it could be. (I'll add tests for these in a seperate PR because chances are there will be more edge cases like that)